packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -54,7 +61,10 @@ export async function createNewFluidFile(
     }
 
     const filePath = newFileInfo.filePath ? encodeURIComponent(`/${newFileInfo.filePath}`) : "";
-    const encodedFilename = encodeURIComponent(`${newFileInfo.filename}.fluid`);
+    const containerPackageName = newFileInfo.containerPackageName;
+    assert(containerPackageName, "File Info should always contain container package name for create new flow");
+    const fileExtension = fileExtensionMap.get(encodeURIComponent(containerPackageName)) ?? ".fluid";
```

Comment:
Is there a reason why we do this encode? Is it to get rid of characters/whitespace somehow? I would imagine both strings to be equivalent

Reply:
I just wanted to be safe. Like what if somebody encoded it before reaching this code? So I wanted to make sure they are in same state. What do you think?

Reply:
I think I will remove it.

Reply:
Hmm, yeah I would remove it, if they're passing around already encoded info where we don't expect them to then we'll have other problems, so better to catch them early with simpler logic/code.

Reply:
Removed.